### PR TITLE
WIP: updated documentation for development on windows

### PR DESCRIPTION
**Pull request by xoriole**

### CODE REVIEW

**Comment by qstokkink** [comment]
ok to test

**Comment by xoriole** [comment]
retest this please

**Comment by xoriole** [comment]
retest this please

File: doc/development/development_on_windows.rst [hunk 1]
```diff
@@ -30,42 +31,57 @@ The first package to be installed is M2Crypto which can be installed using pip (
...
+
+    pip install git+git://github.com/pyqt/python-qt5.git
+
+After installation, check it was correctly installed
+
+.. code-block:: none
+
+    python -c "import PyQt5" # this should work without any error
+
+**Alternatively,** if above steps doesnot work, follow the instructions below.
```
Comment (devos50) [inline doc/development/development_on_windows.rst]: doesnot -> does not

File: doc/development/development_on_windows.rst [hunk 2]
```diff
@@ -30,42 +31,57 @@ The first package to be installed is M2Crypto which can be installed using pip (
...
     nmake
     nmake install
 
-Next, download PyQt5 from `here <https://sourceforge.net/projects/pyqt/files/PyQt5/>`_ and make sure that you download the version that matches with the version of Qt you installed in the previous steps. Extract the binary and compile it:
+Next, download PyQt5 from `here <https://sourceforge.net/projects/pyqt/files/PyQt5/>`__ and make sure that you download the version that matches with the version of Qt you installed in the previous steps. Extract the binary and compile it:
 
 .. code-block:: none
 
-    python configure.py
+    python configure.py --qmake=<qmake_path> --disable=QtNfc --disable=QtBluetooth
```
Comment (devos50) [inline doc/development/development_on_windows.rst]: Please explain why `QtNfc` should be disabled

File: doc/development/development_on_windows.rst [hunk 3]
```diff
@@ -74,13 +90,20 @@ The apsw (Another Python SQLite Wrapper) installer can be downloaded from `GitHu
 libtorrent
 ----------
 
-This package should be compiled from source. First, install Boost which can be downloaded from `SourceForge <http://sourceforge.net/projects/boost/files/boost-binaries/>`_. Make sure to select the latest version and choose the version is compatible with your version of Visual C++ tools (probably msvc-14).
+To install libtorrent, you can simply copy the ``libtorrent.pyd`` file from the github repository `here <https://github.com/Tribler/libtorrent-binaries>`__ and place it inside your python site-packages directory. Then, check check if installation was successful:
```
Comment (devos50) [inline doc/development/development_on_windows.rst]: Use check only once

File: doc/development/development_on_windows.rst [hunk 4]
```diff
@@ -51,7 +51,7 @@ After installation, check it was correctly installed
 
     python -c "import PyQt5" # this should work without any error
 
-**Alternatively,** if above steps doesnot work, follow the instructions below.
+**Alternatively,** if above steps does not work, follow the instructions below.
```
Comment (devos50) [inline doc/development/development_on_windows.rst]: *do not

**Pull request merged**